worldedit-core/src/main/java/com/sk89q/worldedit/internal/expression/ExpressionEnvironment.java

```diff
@@ -36,4 +36,5 @@ public interface ExpressionEnvironment {
 
     int getBlockDataRel(double x, double y, double z);
 
+    ExpressionEnvironment clone();
```

Comment:
Could you put `// FAWE` comments around this method, similar to how it is done in the implementation?